Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:buster as base
+
+ENV PYTHONPATH=${PYTHONPATH}:${PWD} 
+RUN pip3 install poetry gunicorn markupsafe==2.0.1
```

Comment:
This shouldn't be being used to install gunicorn or markupsafe; since we're using poetry as our package manager, pip shouldn't be doing any more than setting up poetry. The other packages will be installed correctly by poetry anyway when we run the `poetry install` step